### ISSUE TITLE
always get `wrong season` in v4

### ISSUE THREAD

**Issue by Fmajor** [Sonarr/Sonarr#6205]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

all release are rejected by the common reason: `wrong season`, even if the Scene Info is identified and the title can pass the parsing test.

All the releases in the interactive search results are rejected by the same common reason: Wrong season. (see fig1 and 2)

fig1
![image](https://github.com/Sonarr/Sonarr/assets/1620897/fac405a5-46c8-42f1-86f4-f2f96a259bd9)

fig2
![image](https://github.com/Sonarr/Sonarr/assets/1620897/a9ce885a-3207-48fe-ac5d-884a832b7a67)

But as you can see in fig3, the second release has Scene Info, and It should not be the wrong season.

fig3
![image](https://github.com/Sonarr/Sonarr/assets/1620897/75b2e84b-8c33-49a6-a26f-1f6411a046a3)

Also see fig4, the title can pass the parsing test and can match to the right series,season and episdes.

fig4
![image](https://github.com/Sonarr/Sonarr/assets/1620897/b78885e4-5a9c-4dda-9c50-d3cf3d2a6a16)

So why do i always get the Wrong season error?

=================================================================
I also test the same indexer in v3
Version
3.0.10.1567
Mono Version
6.12.0.182

using the same indexer, v3 accept the release
![image](https://github.com/Sonarr/Sonarr/assets/1620897/c4abc97b-d63c-4da9-91cb-e6641b0b6c2e)

### Expected Behavior

Release not rejected if Scene Info is identified

### Steps To Reproduce

see above figures

### Environment

```markdown
OS: Gentoo
Docker Install: yes
Sonarr: 4.0.0.731
Package Version: 4.0.0.731-ls11 by linuxserver.io
.NET: Yes (6.0.13)
Using Reverse Proxy: Yes, base Path is `/sonarr`
Database: Sqlite 3.41.2
Browser: Firefox or chrome
```

### What branch are you running?

Develop

### Trace Logs?

update latter

### Anything else?

see above

**Comment by markus101** [comment]
In the v4 instance it appears you have the series type set to `Standard` and in v3 your have it set to Anime, which results in different requirements for the results when processing, which for a single episode search requires the season number of the parsed release to match the season number of the episode, which according to your test parse screenshot it does not.
